operator/src/test/resources/correct-with-proxy-set-podtemplate-keycloak.yml

```diff
@@ -0,0 +1,42 @@
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  name: example-podtemplate-kc
+spec:
+  instances: 1
+  additionalOptions:
+    - name: db
+      value: postgres
+    - name: db-url-host
+      value: postgres-db
+    - name: db-username
+      value: postgres
+    - name: db-password
+      value: testpassword
+    - name: proxy
+      value: reencrypt
```

Comment:
Sorry, I missed this before. This is a pod template test but you are not using the `podTemplate` field to set the proxy value here. Setting proxy via `additionalOptions` is already tested in `KeycloakDeploymentTest`. I think we need to try and set the `KC_PROXY` env var here, instead of using `additionalOptions` for this test to have any value.

Reply:
Due to the fact that the Operator does not support overriding or removing `Env Vars` throughout `PodTemplate`, makes no sense to have such test method. Therefore, I'm removing all this new method from `PodTemplateTest` class.

Thanks.